### DISCUSSION TITLE
Connected in bvm_Network with Metamask but can´t get the tokens on Naka after Claimed and Staked

### DISCUSSION THREAD

**Discussion by maurovsk** [blockscout/blockscout#9594] [Support] [answered]
Hi folks. I signed bvm_network and claimed and staked the tokens always using my metamask wallet. After claimed i didnt save the naka wallet private key.
Now i cant access the wallet.
Do you know if is it possible to solve this issue ?

**Comment by sl1depengwyn** [comment, marked as answer]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
